addons/pos_self_order/controllers/orders.py

```diff
@@ -4,17 +4,17 @@
 import uuid
 from odoo import http, fields, Command
 from odoo.http import request
-from werkzeug.exceptions import NotFound, BadRequest, Unauthorized
+from werkzeug.exceptions import NotFound, Unauthorized
 
 class PosSelfOrderController(http.Controller):
     @http.route("/pos-self-order/process-new-order", auth="public", type="json", website=True)
-    def process_new_order(self, order, access_token, table_identifier):
+    def process_new_order(self, order, access_token):
```

Comment:
Maybe no need to override this. We can derive the table_id from the order. Check to be sure.